generator/osm2meta.cpp

```diff
@@ -24,6 +29,38 @@ constexpr char const * kOSMMultivalueDelimiter = ";";
 auto constexpr kMaxBuildingLevelsInTheWorld = 167;
 auto constexpr kMinBuildingLevel = -6;
 
+// TODO (@gmoryes). Add tool for update this data from URL.
+unordered_map<string, string> const kHumanSpeedToNumber = {
```

Comment:
What do you mean by "human speed"?
What does the "number" mean?

Reply:
Please rename.

Reply:
`kCommonRoadsSpeedToNumericSpeed`
ok?

Reply:
kRoadTypeToSpeedMPS ?

Reply:
okey

Reply:
done